gammapy/maps/axes.py

```diff
@@ -45,6 +45,18 @@ def pix_to_coord(edges, pix, interp="lin"):
     return scale.inverse(interp_fn(pix))
 
 
+PLOT_AXIS_LABEL = {
+    "energy": "Energy",
+    "energy_true": r"E$_{\rm true}$",
```

Comment:
If we are to have a registry, I would go for `True Energy` explicitly

Reply:
Thanks @AtreyeeS . If we choose the text only format, I also like `True Energy`.